The local is returned as C.UTF-8 on musl which should be ok for numeric
operations.

--- ./lib/Smokeping.pm.orig
+++ ./lib/Smokeping.pm
@@ -31,7 +31,8 @@
 my $xssBadRx = qr/[<>%&'";]/;
 
 $ENV{'LC_NUMERIC'}='C';
-if (setlocale(LC_NUMERIC,"") ne "C") {
+my $lc_numeric = setlocale(LC_NUMERIC,"");
+if ($lc_numeric ne "C" && ! $lc_numeric =~ m/^C\./) {
     if ($ENV{'LC_ALL'} eq 'C') {
         # This has got to be a bug in perl/mod_perl, apache or libc
         die("Your internalization implementation on your operating system is "
